src/airodump-ng.c: In function 'main':
src/airodump-ng.c:2124: warning: pointer targets in passing argument 1 of 'fopen' differ in signedness
src/kstats.c: In function 'main':
src/kstats.c:264: warning: pointer targets in passing argument 1 of 'sscanf' differ in signedness
test/makeivs.c: In function 'main':
test/makeivs.c:26: warning: pointer targets in passing argument 1 of 'sscanf' differ in signedness
diff -Naur aircrack-ng-0.6.orig/src/airodump-ng.c aircrack-ng-0.6/src/airodump-ng.c
--- aircrack-ng-0.6.orig/src/airodump-ng.c	2006-06-23 20:21:29.000000000 +0200
+++ aircrack-ng-0.6/src/airodump-ng.c	2006-07-10 00:14:28.000000000 +0200
@@ -2121,7 +2121,7 @@
         G.is_madwifing = 1;
         memset(buffer,0, 65536);
         sprintf((char *) buffer, "/proc/sys/net/%s/dev_type", iface);
-        FILE * f = fopen(buffer,"w");
+        FILE * f = fopen((char*)buffer,"w");
         if (f != NULL) {
             char * madwifiPrism2 = "802\n";
             fprintf(f, madwifiPrism2);
diff -Naur aircrack-ng-0.6.orig/src/kstats.c aircrack-ng-0.6/src/kstats.c
--- aircrack-ng-0.6.orig/src/kstats.c	2006-04-13 01:24:03.000000000 +0200
+++ aircrack-ng-0.6/src/kstats.c	2006-07-10 00:15:36.000000000 +0200
@@ -261,7 +261,7 @@
     buffer[1] = s[1];
     buffer[2] = '\0';
 
-    while( sscanf( buffer, "%x", &n ) == 1 )
+    while( sscanf( (char*)buffer, "%x", &n ) == 1 )
     {
         if( n < 0 || n > 255 )
         {
diff -Naur aircrack-ng-0.6.orig/test/makeivs.c aircrack-ng-0.6/test/makeivs.c
--- aircrack-ng-0.6.orig/test/makeivs.c	2006-04-13 01:24:08.000000000 +0200
+++ aircrack-ng-0.6/test/makeivs.c	2006-07-10 00:16:37.000000000 +0200
@@ -23,7 +23,7 @@
     buffer[1] = s[1];
     buffer[2] = '\0';
 
-    while( sscanf( buffer, "%x", &n ) == 1 )
+    while( sscanf( (char*)buffer, "%x", &n ) == 1 )
     {
         if( n < 0 || n > 255 )
         {
